features/step_definitions/basic_steps.rb

```diff
@@ -36,3 +36,11 @@
   user = User.find_by(email: email)
   login_as(user, scope: :user)
 end
+
+Then(/^I am on the index booking page$/) do
+  visit bookings_path
+end
+
+Given(/^the following facility exist$/) do
+  create(:facility)
+end
```

Comment:
Dudes, no. This can be `Given there is a facility named "Laundry Room"` then create the facility with that name.